### ISSUE TITLE
DE10-Nano: the poweroff command is not working (reboots instead)

### ISSUE THREAD

**Issue by t-legiaw** [robseb/rsyocto#6]
Hi Robin,

the **poweroff** command is not working. Instead of halting the system it reboots.

env ...

- Board: DE10-Nano
- SD Card Image: rsYocto Vers. 1.03 (build 10.01.2020)

for details see attached poweroff_log.txt

-
[poweroff_log.txt](https://github.com/robseb/rsyocto/files/4179486/poweroff_log.txt)
 Hans

**Comment by robseb** [comment]
Hi Hans,

the shut down should work properly.
However, in this case the enabled WatchDog timer is the problem.
After the system shut down successfully no one resets the watchdog timer.
Then restart the Watchdog timer the Linux aigan.
I decided to enable the watchdog to equip the Linux for the typical embedded usage. 
This development boards have no power management controller, that an pythical shutdown allows. That would solve the problem.

Thanks for work with rsYocto and for your iusse request.
Robin

**Comment by t-legiaw** [comment]
Thanks for explanation Robin!

I am using poweroff/halt only to ensure that the sdcard is cleanly unmounted.

But if this is by intention (or by design or too difficult to fix) then I am fine with it ;-)

- Hans

**Comment by Andy2No** [comment]
Is there a better way than simply turning it off?

I tried;

shutdown -h now

then just as I was about to turn off the power supply, I realised it was rebooting.  I'm trying rsYocto with a DE0-Nano-SoC, and logging in from Windows with PuTTy.

Is there anyway to stop the watchdog timer, so it shuts down and stays shut down?  Isn't there a risk of the microSD card getting corrupted if we simply turn it off?
